Mathlib/Algebra/Algebra/NonUnitalSubalgebra.lean

```diff
@@ -399,6 +401,16 @@ theorem gc_map_comap (f : F) :
 theorem mem_comap (S : NonUnitalSubalgebra R B) (f : F) (x : A) : x ∈ comap f S ↔ f x ∈ S :=
   Iff.rfl
 
+@[simp]
+theorem comap_toSubmodule (S : NonUnitalSubalgebra R B) (f : F) :
+    (comap f S).toSubmodule = Submodule.comap (LinearMapClass.linearMap f) S.toSubmodule :=
+  rfl
```

Comment:
I assume this lemma isn't generated by simps because we are missing the `toSubmodule` projection hasn't been declared to simps. Would you mind trying to do so by using `initialize_simps_projection` on `NonUnitalSubalgebra`?